backend/Perflow/Migrations/20210803162535_RunSeeding.cs

```diff
@@ -0,0 +1,268 @@
+                    { 2, 2, 2, 0, new DateTimeOffset(new DateTime(2020, 9, 9, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 3, 0, 0, 0)), 4, null, false, "https://images.saymedia-content.com/.image/t_share/MTc0NDkxNzgyMzYzNDg5NjQw/vinyl-to-paper-how-to-write-an-album-review.jpg", "Mind Games" },
+                    { 3, 2, 2, 0, new DateTimeOffset(new DateTime(2020, 9, 9, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 3, 0, 0, 0)), 2, null, false, "https://images.saymedia-content.com/.image/t_share/MTc0NDkxNzgyMzYzNDg5NjQw/vinyl-to-paper-how-to-write-an-album-review.jpg", "Imagine" },
+                    { 4, 3, 2, 0, new DateTimeOffset(new DateTime(2020, 9, 9, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 3, 0, 0, 0)), 3, null, false, "https://images.saymedia-content.com/.image/t_share/MTc0NDkxNzgyMzYzNDg5NjQw/vinyl-to-paper-how-to-write-an-album-review.jpg", "Only People" }
+                });
+
+            migrationBuilder.InsertData(
+                table: "PlaylistSong",
+                columns: new[] { "Id", "CreatedAt", "PlaylistId", "SongId" },
+                values: new object[,]
+                {
+                    { 1, new DateTimeOffset(new DateTime(2020, 10, 19, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 3, 0, 0, 0)), 1, 1 },
+                    { 4, new DateTimeOffset(new DateTime(2020, 2, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 2, 0, 0, 0)), 2, 1 },
+                    { 2, new DateTimeOffset(new DateTime(2020, 10, 19, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 3, 0, 0, 0)), 1, 2 },
+                    { 5, new DateTimeOffset(new DateTime(2020, 2, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 2, 0, 0, 0)), 2, 2 },
+                    { 7, new DateTimeOffset(new DateTime(2020, 12, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 2, 0, 0, 0)), 3, 2 },
+                    { 3, new DateTimeOffset(new DateTime(2020, 10, 19, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 3, 0, 0, 0)), 1, 3 },
+                    { 6, new DateTimeOffset(new DateTime(2020, 2, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 2, 0, 0, 0)), 2, 3 },
+                    { 8, new DateTimeOffset(new DateTime(2020, 12, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 2, 0, 0, 0)), 3, 3 },
+                    { 9, new DateTimeOffset(new DateTime(2020, 12, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 2, 0, 0, 0)), 3, 4 }
+                });
+
+            migrationBuilder.InsertData(
+                table: "SongReactions",
+                columns: new[] { "Id", "SongId", "UserId" },
+                values: new object[,]
+                {
+                    { 1, 1, 1 },
+                    { 2, 3, 1 },
+                    { 3, 4, 1 }
+                });
+        }
+
```

Comment:
We don't need a migration for seeding. As I know there is a script for seeding.
Ask @krolebord more about it.